extra/man/alacritty.5.scd

```diff
@@ -52,16 +67,21 @@ Windows: _"powershell"_
 
 	Default: _true_
 
+Default:
+
+	live_config_reload = _true_++
+working_directory = _"None"_
+
 # ENV
 
 All key-value pairs in the *env* section will be added as environment variables
 for any process spawned by Alacritty, including its shell. Some entries may
 override variables set by alacritty itself.
 
-```
-[env]
-TERM = "alacritty"
-```
+Example:
+
+	*[env]*++
+TERM = _"alacritty"_
```

Comment:
Overriding `TERM` is usually a bad idea. I wonder if we could use a better example variable here? Maybe something from winit?

Reply:
That's a good idea.